registry.json

```diff
@@ -2405,6 +2412,7 @@
     "curve": "secp256k1",
     "publicKeyType": "secp256k1Extended",
     "hrp": "evmos",
+    "publicKeyHasher": "keccak256",
```

Comment:
This needs to add to all EVM items, not just Evmos

Reply:
Ethereum implementation does not use publicKeyHasher field, now only Cosmos implementation was extended to use this field.  Other than Cosmos, only Bitcoin uses it.

Reply:
Yes, but this issue is existing for a while: developer might call `TW::publicKeyHasher(TWCoinTypeEthereum)` and get the default value `sha256ripemd`, so I think it's a good time to fix it

Reply:
Sure, done.